src/components/ListItem/ListItem.scss

```diff
@@ -33,18 +36,9 @@ $big-image-size: 96px;
 .moonstone-listItem_iconEnd {
     display: flex;
     align-items: center;
-    min-width: 16px;
```

Comment:
Why do you remove the `min-width` ?

Reply:
Removing it did not cause any incident on related components